app/helpers/report_helper/timer.rb

```diff
@@ -0,0 +1,70 @@
+ReportHelper::Timer = Struct.new(
+  :typ,
+  :months,
+  :weeks,
+  :days,
+  :hours,
+  :start_date,
+  :start_hour,
+  :start_min,
+) do
+  def initialize(typ = nil, months = 1, weeks = 1, days = 1, hours = 1, start_date = nil, start_hour = '00', start_min = '00')
+    super
+  end
```

Comment:
The struct buys me something. I think I would need to be more verbose when defining constructor with defaults. The only problem I see is that I cannot include the mixin with these timezone methods. That can be workarounded by re-open of the struct.